app/src/main/java/io/github/droidkaigi/confsched/model/Page.java

```diff
@@ -21,7 +22,7 @@ public Fragment createFragment() {
             return SessionsFragment.newInstance();
         }
     },
-    MY_SCHEDULE(R.id.nav_my_schedule, R.string.my_schedule, false, MyScheduleFragment.class.getSimpleName()) {
+    MY_SCHEDULE(R.id.nav_my_schedule, R.string.my_schedule, false, MyScheduleFragment.class.getSimpleName(), R.color.amber500, R.color.amber600) {
```

Comment:
Could you create new color in colors.xml?

```xml
<color name="bluegrey500">#607D8B</color>
<color name="bluegrey600">#546E7A</color>
#607D8B
```

Because `amber` color is already used as specific category color.